src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -256,18 +257,19 @@ private JToken LoadXrefProperty(Document file, Context context, SourceInfo<strin
                     errors.AddRange(markupErrors);
 
                     // todo: use BuildPage.CreateHtmlContent() when we only validate markdown properties' bookmarks
-                    return (errors, HtmlUtility.LoadHtml(html).PostMarkup(context.Config.DryRun).WriteTo());
+                    return (errors, html);
```

Comment:
is `stripTags` fast enough for `DryRun` now?

Reply:
Yes, I think so.